xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+  type="text/xml"
+  title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!--
+FATE#320599: Pacemaker Remote
+-->
+<?provo dirname="nfs_quick/"?>
+<article version="5.0" xml:lang="en" xml:id="art_sle_ha_pmremote"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:dm="urn:x-suse:ns:docmanager"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>&paceremote;</title>
+  <subtitle>&productname; &productnumber;</subtitle>
+  <info>
+   <productnumber>&productnumber;</productnumber>
+   <productname>&productname;</productname>
+   <date><?dbtimestamp format="m/d/Y X"?></date>
+   <abstract>
+    <para>
+     This document guides you through the setup of a cluster with virtual
+     guest nodes, managed by &pace; and &pmremote;.
+     <emphasis>Remote</emphasis> in this context means not physical proximity,
+     but <quote>non-membership</quote> of a cluster.
+    </para>
+   </abstract>
+  </info>
+
+ <sect1 xml:id="sec.ha.pmremote.usage">
+  <title>Usage Scenario</title>
+  <para>
+   The procedures in this document will lead to a minimal setup of a cluster
+   with the following properties:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <remark>toms 2017-03-23: Is this supported from 12 GA, SP1, or SP2?</remark>
```

Comment:
Definitely since SP1 at least.  Not sure about GA - I suspect that it was but there were quite a few bugs in the early versions which I'm not sure got fixed for GA.  @gao_yan would know.